### ISSUE TITLE
Sporadic "No Network Connection" Issues

### ISSUE THREAD

**Issue by Qwynn** [yeriomin/YalpStore#89]
I'm running Yalp on my AOSP 6.0.1 (No GAAPS) Nexus 5 without issue.

However, I'm encountering repeated network issues on my AOSP 6.0.0 (No GAAPS) Nexus 10. Despite having a solid network connection, Yalp continually insists I don't. Attempting to use Yalp is a constant struggle of refreshing searches and attempting to get app details while hoping Yalp properly sees my connection.

I was surprised I even managed to sign in to use, and that alone took an hour of attempts.

I'm not really knowledgeable on the coding side of things for Android, so if there's anything I can provide which would help let me know.

**Comment by yeriomin** [comment]
There is a new version out on f-droid. Please check if it helps. I added some debug logging too, so logs are going to be a bit more useful.

**Comment by yeriomin** [comment]
No one else reported anything similar in a month, so I'm closing this.

**Comment by bentosilveira** [comment]
I was having this issue with Resurrection Remix 6.0 (Oreo 8.1 based), no GAPPS installed. Only, it was not intermittent, it was always. I ran a logcat while using Yalp and found that it attempts to connect to android.clients.google.com, which I have blocked due to privacy concerns. So, this issue could be related to A) Adblocking/Privacy apps blocking Google a little too much or B) User paranoia. After I briefly white-listed that address, it worked perfectly.

I'm commenting on this closed issue just so you know one of the possible causes for this, in case it re-appears.
